From: Vincent Davis Jr <vince@underview.tech>
Date: Fri, 07 Jan 2022 07:10:47 PM CST
Subject: [PATCH] Fix numeric limits not a member

* Fixes bellow compiler issue:
  ../../git/modules/demux/adaptive/playlist/SegmentInformation.cpp:397:23: error: 'numeric_limits' is not a member of 'std'
  397 |     if(number == std::numeric_limits<uint64_t>::max())

Upstream-status: Pending

Signed-off-by: Vincent Davis Jr <vince@underview.tech>
diff --git a/modules/demux/adaptive/playlist/SegmentInformation.cpp b/modules/demux/adaptive/playlist/SegmentInformation.cpp
index 344e155c7..8eeb05439 100644
--- a/modules/demux/adaptive/playlist/SegmentInformation.cpp
+++ b/modules/demux/adaptive/playlist/SegmentInformation.cpp
@@ -34,6 +34,7 @@

 #include <algorithm>
 #include <cassert>
+#include <limits>

 using namespace adaptive::playlist;

diff --git a/modules/demux/hls/playlist/Parser.cpp b/modules/demux/hls/playlist/Parser.cpp
index 67110e4cd..743969922 100644
--- a/modules/demux/hls/playlist/Parser.cpp
+++ b/modules/demux/hls/playlist/Parser.cpp
@@ -42,6 +42,7 @@
 #include <map>
 #include <cctype>
 #include <algorithm>
+#include <limits>

 using namespace adaptive;
 using namespace adaptive::playlist;
